.travis.yml

```diff
@@ -5,15 +5,22 @@ python:
   - "pypy"
 before_install:
   - "pushd .."
-  - "git clone https://github.com/pulp/pulp.git --branch 2.5-release"
+  - "echo 'Remove this'"
+  - "sudo apt-get install swig"
+  - "git clone https://github.com/rbarlow/pulp.git --branch 278"
+  - "pulp/manage_setup_pys.sh develop"
+  - "pulp/manage_setup_pys.sh develop --uninstall"
+  - "rm -rf pulp"
+  - "git clone https://github.com/pulp/pulp.git --branch master"
   - "git clone https://github.com/pulp/nectar.git"
   - "pushd nectar"
   - "git checkout python-nectar-1.1.6-1"
   - "popd"
   - "popd"
 install:
+  # This is needed to build M2Crypto
+  - "sudo apt-get install swig"
```

Comment:
On 02/20/2015 05:45 PM, Austin wrote:
> This is here twice.

This is also on purpose. I needed it to be in that block above that will
get removed, so I wanted to make sure it was here too so that it doesn't
get removed when I fix the above block.